config/development-cass.yaml

```diff
@@ -1,3 +1,14 @@
+otel:
```

Comment:
Why is it on top? Can you move it down? Also should go to other configs too. I recently synced them all.

Reply:
Removed from all configs so this is opt-in.